interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -488,6 +527,83 @@ function set_joint_target_positions(obj,handles,thetas,opmode)
                 thetas(joint_index) = double(tmp);
             end
         end
+
+        %% Get joint velocities
+        function [joint_velocities,retval]=get_joint_velocities(obj,handles,opmode)
+            %% Get joint velocities
+            %%  >> joint_names = {'redundantRob_joint1','redundantRob_joint2','redundantRob_joint3','redundantRob_joint4','redundantRob_joint5','redundantRob_joint6','redundantRob_joint7'};
```

Comment:
Is this comment necessary?

Reply:
Not really. I implemented the new methods and their respective documentation following the current style of the class `DQ_VrepInterface`.  That is the reason for those comments. However, I removed them.